### PR TITLE
Add XI-IATI-AGR and XI-IATI-OCHASDC

### PR DESCRIPTION
**Pull request by amy-silcock**
Adding two new organisations:
https://discuss.iatistandard.org/t/approved-request-for-new-org-id-xi-iati-ochasdc/1378
https://discuss.iatistandard.org/t/new-organisation-identifier-xi-iati-agr/1417

### CODE REVIEW

File: xml/IATIOrganisationIdentifier.xml [hunk 1]
```diff
@@ -48,6 +48,16 @@
...
         </codelist-item>
+        <codelist-item>
+            <code>XI-IATI-AGR</code>
+            <name>
+                <narrative>AgResults</narrative>
+            </name>
+            <description>
+                <narrative>AgResults is a $122 million collaborative initiative between the governments of Australia, Canada, the United Kingdom, the United States, and the Bill and Melinda Gates Foundation to incentivize the private sector to overcome market barriers and develop solutions to food security and agricultural challenges that disproportionately affect people living in poverty. The initiative designs and implements agriculture-focused prize competitions, also referred to as pay-for-results or pull mechanisms, which are innovative development finance programs that engage the private sector to work towards a defined goal to receive a monetary award.</narrative>
+            </description>
+            <url>https://http://agresults.org/</url>
```
Comment (andylolz) [inline xml/IATIOrganisationIdentifier.xml]: typo: repeated protocol

Comment (andylolz) [reply xml/IATIOrganisationIdentifier.xml]: of the two, https is probably preferable.

File: xml/IATIOrganisationIdentifier.xml [hunk 2]
```diff
@@ -201,6 +211,16 @@
...
         </codelist-item>
+        <codelist-item>
+            <code>XI-IATI-OCHASDC</code>
+            <name>
+                <narrative>United Nations Office for the Coordination of Humanitarian Affairs - Specially-Designated Contributions</narrative>
+            </name>
+            <description>
+                <narrative>The United Nations Office for the Coordination of Humanitarian Affairs (OCHA) is the part of the United Nations Secretariat responsible for bringing together humanitarian actors to ensure a coherent response to emergencies. OCHA’s activities are funded mostly through voluntary contributions from UN Member States, these contributions are reported under the IATI organisation ID XM-DAC-41127. OCHA also administers several Specially-Designated Contributions (SDCs) for projects implemented by third parties. Activities funded through SDCs will be reported under the IATI organisation ID XI-IATI-OCHASDC.</narrative>
+            </description>
+            <url>https://www.unocha.org/</url>
```
Comment (andylolz) [inline xml/IATIOrganisationIdentifier.xml]: Does a more specific URL exist?

Comment (andylolz) [reply xml/IATIOrganisationIdentifier.xml]: (SDC-specific, I mean.)

Comment (Ocre42) [reply xml/IATIOrganisationIdentifier.xml]: Maybe https://www.unocha.org/our-work (?)

**Pull request merged**